client/src/ledger_app_clients/ethereum/client.py

```diff
@@ -271,3 +270,7 @@ def provide_nft_metadata(self,
                                                                     key_id,
                                                                     algo_id,
                                                                     sig))
+
+
+    def external_plugin_setup(self, plugin_name: str, contract_address: bytes, method_selelector: bytes, sig: bytes):
```

Comment:
For now I would name it `set_external_plugin`, to keep it in line with the `set_plugin` [here](https://github.com/LedgerHQ/app-ethereum/blob/171b31d3a325d10da16dde63e9924e7ae9d48d31/client/src/ledger_app_clients/ethereum/client.py#L210).
I would also give the signature parameter a default value of `None` just like the other one.

Reply:
ok